docs/pages/build-reference/infrastructure.mdx

```diff
@@ -366,3 +366,8 @@ iOS builder VMs run on Mac mini hosts in an isolated environment. Every build ge
 - node-gyp 10.0.1
 
 </Collapsible>
+
+### Supported Xcode versions
+
+We aim to support all of the stable Xcode releases that when used during the build process allow you to submit your app to the App Store Connect.
+This usually means that we support the latest stable Xcode version and the previous one (until the new [minimal Xcode version requirement](https://developer.apple.com/news/upcoming-requirements/?id=04292024a) is introduced by Apple).
```

Comment:
```suggestion
We aim to support all stable Xcode releases that allow you to submit your app to the App Store Connect when used during the build process.

This usually means that we support the latest stable Xcode version and the previous one (until the new [minimal Xcode version requirement](https://developer.apple.com/news/upcoming-requirements/?id=04292024a) is introduced by Apple).
```

I think we can improve the first sentence here for better readability. Without adding the pause in the previous version, it was hard to read.